src/os/bluestore/BlueStore.cc

```diff
@@ -8550,8 +8550,10 @@ void BlueStore::_kv_sync_thread()
 	} else if (deferred_aggressive) {
 	  force_flush = true;
 	}
-      } else
-	force_flush = true;
+      } else {
+      	if(aios || !deferred_done.empty())
```

Comment:
<pre>
if (aios || !deferred_done.empty()) {
  force_flush = true;
}
</pre>
￼

Reply:
can you add an else { dout(20) << __func__ << " skipping flush (no aios, no deferred_done)" << dendl; } too?

Reply:
@liewegas fixed.